Simplenote/StorageSettings.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+
+@objcMembers
+class StorageSettings: NSObject {
+
+    /// URL for the managed object model resource
+    ///
+    let modelURL: URL = {
+        Bundle.main.url(forResource: Constants.resourceName, withExtension: Constants.resourceType)!
+    }()
+    /// In app core data storage URL
+    ///
+    let legacyStorageURL: URL = {
+        FileManager.default.documentsURL.appendingPathComponent(Constants.sqlFile)
+    }()
```

Comment:
Same note @eshurakov mentioned for `modelURL` (same applies for the other getters!)